src/app/accessRequests/views/reviewOrganisationRequest.ejs

```diff
@@ -96,7 +96,13 @@
 
                     </fieldset>
                 </div>
-
+                <div class="govuk-warning-text">
+                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                    <strong class="govuk-warning-text__text">
+                      <span class="govuk-warning-text__assistive">Warning</span>
+                      By approving this request, the access for the <%= locals.request.usersName %> will change immediatley.
```

Comment:
`By approving this request, the access for the <%= locals.request.usersName %> will change immediatley.`
- remove the second **the** 
- correct spelling mistake for `immediatley` to `immediately`

Reply:
removed and fixed ta